package.json

```diff
@@ -21,17 +21,17 @@
   "main": "./lib/union.js",
   "license": "MIT",
   "scripts": {
-    "test": "node-gyp rebuild --directory test && mocha -gc --reporter spec"
+    "test": "node-gyp rebuild --directory test && mocha --expose-gc --reporter spec"
   },
   "dependencies": {
-    "debug": "^3.1.0"
+    "debug": "^4.3.1",
+    "node-gyp-build": "^4.2.1"
```

Comment:
Is this also supposed to be a dev dependency?

Reply:
Yes, thank you, fixed!